tests/Doctrine/Tests/ORM/Mapping/ClassMetadataInfoTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Doctrine\Tests\ORM\Mapping;
+
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Doctrine\ORM\Mapping\ClassMetadataInfo;
+use PHPUnit\Framework\TestCase;
+
+class ClassMetadataInfoTest extends TestCase
+{
+    /**
+     * @group legacy
+     * @expectedDeprecation Doctrine\ORM\Mapping\ClassMetadataInfo is deprecated since 2.x and will be removed in 3.0. Use Doctrine\ORM\Mapping\ClassMetadata instead.
+     */
+    public function testTheClassIsDeprecated()
+    {
+        class_exists(ClassMetadataInfo::class);
```

Comment:
Should at least has assertTrue(). Maybe instantiating ReflectionClass would be better as it would throw i.e. once CMI BC is dropped.

Reply:
Once CMI BC is dropped, the class will no longer exist, and this will throw too. Even without the `assertTrue`, right? I mean there is the `@expectedDeprecation`, and it will no longer be satisfied, I don't know why you want me to add `assertTrue`. I'll add it but I don't understand what the point is.

Reply:
Doesn't matter really, just thought it may be easier to see failed assertion first than _no deprecation thrown_ or whatever that message looks like.